unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/DecentralandEntity.cs

```diff
@@ -180,6 +180,26 @@ public void RemoveSharedComponent(System.Type targetType, bool triggerDettaching
             }
         }
 
+        public BaseComponent GetBaseComponent(CLASS_ID_COMPONENT componentId)
```

Comment:
Most of times this method is used followed by a `null` check. We could convert it to a TryGet method

```
        public bool TryGetBaseComponent(CLASS_ID_COMPONENT componentId, out BaseComponent component)
        {
        	return components.TryGetValue(componentId, out component);
        }
```